### PR TITLE
Bye bye Madara

### LINKED ISSUE #5686: [Source Request] <MangaEffect>
**PLEASE READ THIS**

I acknowledge that:

- I have checked that the extension does not already exist:
  - In the list of existing extensions in the application
  - In the multi-source spreadsheet: https://tachiyomi.org/extensions
  - By searching the GitHub repository for the extension and verify it does not appear in the code base
- I have searched the existing GitHub issues and this extension does **NOT** have an open request
- I will fill out the title and the information in this template

Note that the issue will be automatically closed if you do not fill out the title or requested information.

---

## Source website
- Name: MangaEffect
- Link: MangaEffect.com
- Language(s): English

### LINKED ISSUE #5712: [Bug] Arang Scans: not pulling any chapter info
## Device information
- Tachiyomi version: 0=10.8
- Android version: 11/ OneUI 3.1

## Source information
- Name: Arang Scans
- Extension version: 1.2.1

## Steps to reproduce
1. Open Arang Scans thru extension using the normal or latest view
2. Tap on any manga and wait for the error message
3. Open the manga in webview and see that there are chapters

### Expected behavior
Chapter list should have come up

### Actual behavior
"No chapters found"

### LINKED ISSUE #5736: [Source Request] <Manga Great>
## Source website
- Name: Manga Great
- Link: https://mangagreat.com/
- Language(s): English

### PR DESCRIPTION
**Pull request by Riztard**
remove madara
add madara source to splitted multisrc
fix arangscans Closes #5712
add mangaEffect Closes #5686
add mangaGreat Closes #5736

### CODE REVIEW

**Comment by arkon** [comment]
Build failed, FYI.

**Comment by Riztard** [comment]
any way to know which park causing it to failed?

**Comment by arkon** [comment]
Looks like it died while lining Toonily.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -78,8 +78,149 @@ class MadaraGenerator : ThemeSourceGenerator {
...
+            SingleLang("Manga SY", "https://www.mangasy.com", "en"),
+            SingleLang("MangaTeca", "https://www.mangateca.com", "pt-BR"),
+            SingleLang("Manga Turf", "https://mangaturf.com", "en"),
+            SingleLang("MangaTX", "https://mangatx.com", "en"),
+            SingleLang("Manga Weebs", "https://mangaweebs.in", "en"),
+            SingleLang("MangaWT", "https://mangawt.com", "tr"),
+            SingleLang("MangaYaku", "https://mangayaku.my.id", "id"),
+            SingleLang("MangaYosh", "https://mangayosh.xyz", "id"),
+            SingleLang("Mangazuki.club", "https://mangazuki.club", "ja", className = "MangazukiClubJP"),
+            SingleLang("Mangazuki.club", "https://mangazuki.club", "ko", className = "MangazukiClubKO"),
```
Comment (AriaMoradi) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:133]: turn this into a `MultiLang`

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 2]
```diff
@@ -78,8 +78,149 @@ class MadaraGenerator : ThemeSourceGenerator {
...
+            SingleLang("Nazarick Scans", "https://nazarickscans.com", "en"),
+            SingleLang("NeatManga", "https://neatmanga.com", "en"),
+            SingleLang("NekoBreaker", "https://nekobreaker.com", "pt-BR"),
+            SingleLang("NekoScan", "https://nekoscan.com", "en"),
+            SingleLang("Neox Scanlator", "https://neoxscans.com", "pt-BR"),
+            SingleLang("Night Comic", "https://www.nightcomic.com", "en"),
+            SingleLang("Niji Translations", "https://niji-translations.com", "ar"),
+            SingleLang("Ninjavi", "https://ninjavi.com", "ar"),
+            SingleLang("NTS Void Scans", "https://ntsvoidscans.com", "en"),
+            SingleLang( "Off Scan", "https://offscan.top", "pt-BR"),
```
Comment (AriaMoradi) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:169]: lint?

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

**Comment by AriaMoradi** [comment]
The rest looks fine to me.

**Pull request merged**